pkg/api/blocks/v1.go

```diff
@@ -84,9 +86,18 @@ func (bapi *BlocksAPI) Register(r *route.Router, tracer opentracing.Tracer, logg
 
 	r.Get("/blocks", instr("blocks", bapi.blocks))
 	r.Post("/blocks/mark", instr("blocks_mark", bapi.markBlock))
+	r.Get("/flags", instr("flags_disableAdminOperations", bapi.disableAdminOperationsHandler))
+}
+
+func (bapi *BlocksAPI) disableAdminOperationsHandler(r *http.Request) (interface{}, []error, *api.ApiError, func()) {
+	response := map[string]bool{"disableAdminOperations": bapi.disableAdminOperations}
+	return response, nil, nil, func() {}
 }
 
 func (bapi *BlocksAPI) markBlock(r *http.Request) (interface{}, []error, *api.ApiError, func()) {
+	if bapi.disableAdminOperations {
```

Comment:
when i will will use /status/flags endpoint directly, I will have to get the flagRes here also to restrict the use of API. is  this a valid approach? @saswatamcode